src/kqueue.rs

```diff
@@ -221,7 +221,7 @@ impl Events {
             data: 0,
             udata: 0 as _,
         };
-        let list = vec![ev; 1000].into_boxed_slice();
```

Comment:
I'm not sure why the old code used 1000 instead of 1024.

Reply:
It would be a mis-optimization, if I had to guess.